tools/tf2openapi/types/tfmetagraph_test.go

```diff
@@ -188,3 +190,40 @@ func TestNewTFMetaGraphWithErrSignatureDef(t *testing.T) {
 	_, err := NewTFMetaGraph(metaGraphPb)
 	g.Expect(err).Should(gomega.Not(gomega.BeNil()))
 }
+
+func TestTFMetaGraphTypical(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+	tfMetaGraph := expectedTFMetaGraph()
+	expectedSchema := &openapi3.Schema{
+		Type: "object",
+		Properties: map[string]*openapi3.SchemaRef{
+			"instances": {
+				Value: &openapi3.Schema{
+					Type: "array",
+					Items: &openapi3.SchemaRef{
+						Value: &openapi3.Schema{
+							Type:     "array",
+							MaxItems: func(u uint64) *uint64 { return &u }(3),
+							MinItems: 3,
+							Items: &openapi3.SchemaRef{
+								Value: &openapi3.Schema{
+									Type: "number",
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+	schema, err := tfMetaGraph.Schema("sigDefKey")
+	g.Expect(schema).Should(gomega.Equal(expectedSchema))
+	g.Expect(err).To(gomega.BeNil())
+}
+
+func TestTFMetaGraphMissingSigDef(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+	tfMetaGraph := expectedTFMetaGraph()
+	_, err := tfMetaGraph.Schema("missingSigDefKey")
+	g.Expect(err).To(gomega.Not(gomega.BeNil()))
```

Comment:
Can you make the Error message constant and verify the error message here?

Check out how we do the validators in /pkg/api